documentation/staging/content/userguide/managing-domains/upgrade-domain-resource/_index.md

```diff
@@ -0,0 +1,56 @@
++++
+title = "Upgrade Domain resource"
+date = 2019-02-23T16:43:45-05:00
+weight = 2.5
+pre = "<b> </b>"
++++
+
+### Contents
+
+ - [Automated upgrade of WKO 3.x domain resource](#automated-upgrade-of-wko-3.x-domain-resource)
+ - [Upgrade the WKO 3.x domain resource manually](#upgrade-the-wko-3.x-domain-resource-manually)
+
+### Automated upgrade of WKO 3.x domain resource
+The 4.0 Operator provides a seamless upgrade of the Domains with 3.x (V8 schema) domain resources. When you create a Domain using WKO 3.x (V8 schema) domain resource in a namespace managed by the WKO 4.0, the [WebLogic Domain resource conversion webhook]({{< relref "/userguide/managing-operators/conversion-webhook.md" >}}) performs an automated upgrade of the domain resource to 4.0 schema. The conversion webhook runtime converts the WKO 3.x configuration to the equivalent configuration in WKO 4.0. Similarly, when [upgrading the Operator version from 3.x to 4.0]({{< relref "/userguide/managing-operators/installation#upgrade-the-operator" >}}), Domains resources are seamlessly upgraded.
```

Comment:
As above, I think it's confusing to keep talking about 3.x and 4.0 versions of the resources rather than use the API versions. I'd rather see an introduction to 4.0 support for a "v9" Domain resource API version and then talk about "v8" versus "v9".

Reply:
Added a section `Operator 4.0 domain resource API version change` in the beginning and then talk about "v8" and "v9" schema versions. Please let me know if this makes sense and/or if other changes are needed.